eg/Cli.py

```diff
@@ -118,18 +118,45 @@ class args:
             import NamedPipe
 
             if args.restart:
-                import time
-
-                NamedPipe.send_message('eg.app.Exit, ()')
-
-                while ctypes.GetLastError() != 0:
-                    time.sleep(0.1)
-
-                    appMutex = ctypes.windll.kernel32.CreateMutexA(
-                        None,
-                        0,
-                        "Global\\EventGhost:7EB106DC-468D-4345-9CFE-B0021039114B"
-                    )
+                # import time
+
+                try:
+                    if NamedPipe.send_message('eg.document.IsDirty, ()'):
+                        answer = ctypes.windll.user32.MessageBoxA(
+                            0,
+                            'EventGhost cannot restart.        \n\n'
+                            'Save configuration changes?.      \n',
```

Comment:
HA!!  I thought my wording was wrong and you go and lay something on me that is worse off....

    "Configuration contains unsaved changed.\n\n"
    "Do you want to save before continuing?"

LMAO

I will fix the wording in document.py and issue a PR for it. and I will also change the wording to match in this PR